src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -104,48 +104,39 @@ public async Task Build_Valid_SingleFile_WithTemplateSpecReference_ShouldSucceed
         }
 
         [DataTestMethod]
-        [DataRow("br:mcr.microsoft.com/bicep/providers/az", true)]
-        [DataRow("br/public:az", true)]
-        [DataRow("br/contoso:az", true)]
+        [DataRow("br:mcr.microsoft.com/bicep/providers/az", true, LanguageConstants.BicepPublicMcrRegistry)]
+        [DataRow("br/public:az", true, LanguageConstants.BicepPublicMcrRegistry)]
```

Comment:
I can't get this syntax to compile successfully outside of the context of this test. What do I need to do to get this to work?

![provider_statement](https://github.com/Azure/bicep/assets/705500/39005183-e792-4ef0-9a0b-bf927a0ba6ab)

Reply:
resolved offline, bicepconfig.json with proper settings was necessary
```json
{
  "experimentalFeaturesEnabled": {
    "extensibility": true,
    "dynamicTypeLoading": true
  },
  "providerAliases": {
    "br": {
      "public": {
        "registry": "biceptestdf.azurecr.io",
        "modulePath": "bicep/providers"
      }
    }
  }
}
```